### ISSUE TITLE
【L-Softmax】Large-Margin Softmax Loss for Convolution Neural Networks | Insight

### ISSUE THREAD

**Issue by Walstruzz** [Walstruzz/walstruzz.github.io#3]
https://walstruzz.github.io/%E4%BA%BA%E8%84%B8%E8%AF%86%E5%88%AB/2020/12/20/L-Softmax-Large-Margin-Softmax-Loss-for-Convolution-Neural-Networks/ 

 链接：Paper, Github 这篇文章主要提出了Large-Margin Softmax Loss，是后续NormFace、SphereFace、CosFace、ArcFace等一系列文章的基础。 动机 作者发现当前的CNN学习目标倾向于最大化类内紧凑(intra-class compactness)和类间稀疏性(inter-class separability)，比如Triplet Loss，而Triplet Loss计算复杂度较高，不利于大数据训练，而如此简单好用又如此合适的Softmax Loss却少有这方面的研究。Softmax Loss的格式为 \[L=\frac{1}{N}\Sigma_{i}{L_{i}}=\frac{1}{N}\Sigma_{i}{-\log{(\frac{e^{f_{y_{i}}}}{\Sigma_{j}{e^{f_{j}}}})}}\]
